frontend/src/components/AddMaterialDialog.jsx

```diff
@@ -54,7 +54,7 @@ export default function AddMaterialDialog({ open, onClose, onSave }) {
 			return;
 		}
 
-		onSave({ name, description, artifactTypeId }); // Call onSave callback with the new material data
+		onSave({ name, description, artifact_type:artifactTypeId }); // Call onSave callback with the new material data
```

Comment:
why do we need to do this now? come on man

Reply:
React prefers CammelCase over snake_case, using a better naming scheme will have to wait until we finish out #28 and use components as objects not functions.